doc-content/enterprise-only/installation/installer-run-cli-proc.adoc

```diff
@@ -98,6 +99,18 @@ Would you like to generate an automatic installation script and properties file?
 ----
 . Enter `y` to create an XML file that contains the installation data, or `n` to complete the installation. If you enter `y`, you are prompted to specify a path for the XML file.
 . Enter a path or press the Enter key to accept the suggested path.
++
+You can use this file to automatically install {PRODUCT} on the same type of server. Note that you must change the `installpath` parameter in the XML file to specify the path of the new server that you want to install {PRODUCT} on. Enter the following command to perform an installation with the XML file:
```

Comment:
Some suggestions, I like this wording but imho we need some adjustments:

Installer will generate two files. One XML file for automating the installation and one variable file that is used to store password for user and other sensitive variables.
The XML file allows you to easily repeat {PRODUCT} installation on the same type of server with same configuration on multiple machines. You can use the click-through installer generate this XML file once and then easily repeat the installation on different machines. It can also be used it on the same machine, but you must change the `<installpath>` parameter in the XML file to point towards new server.
Enter the following command to perform an installation with the XML file: